cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
@psav If we separate it, it would make sense to me to put it in the `BaseProvider`, sure. All providers would have `properties_form` property; some would base it on `_properties_form` and others on `_properties_region`.